Cargo.toml

```diff
@@ -3,11 +3,18 @@ name = "pg_query"
 description = "PostgreSQL parser that uses the actual PostgreSQL server source to parse SQL queries and return the internal PostgreSQL parse tree."
 version = "6.0.0"
 edition = "2021"
-documentation = "https://docs.rs/pg_query/"
-build = "build.rs"
+documentation = "https://docs.rs/pg_query"
 license = "MIT"
-readme = "./README.md"
```

Comment:
The `build` and `readme` settings are unnecessary since the files are using default file names.